drift/views/phony_data.py

```diff
@@ -0,0 +1,24 @@
+PHONY_DATA = {'packages': [{'name': 'gcc',
```

Comment:
A follow up would be to add in the `hosts` section of the schema that identifies each host in the comparison and it's metadata.

Reply:
exactly

Reply:
@ehelms I'll add it in a new PR, I wanted to get this one in so @johnsonm325 can change his URL